api/auth/auth.service.js

```diff
@@ -12,22 +12,18 @@ export function verifyToken(token, repository = Repository) {
   const userRepository = repository.users;
   return new Promise((resolve, reject) => {
     jwt.verify(token, config.secrets.session, (err, user) => {
-
-
       if (err) {
         reject(err.message);
         return;
       }
 
-      userRepository
-        .findById(user.id)
-        .then(foundUser => {
-          if (!foundUser) {
-            reject(new Error('User not found'));
-          } else {
-            resolve(foundUser);
-          }
-        });
+      userRepository.findById(user.id).then(foundUser => {
```

Comment:
re style: surprised that the method chains are all in-line here, whereas you're pretty consistent with newlines for the rest of this file.